servers/rendering/rendering_server_default.h

```diff
@@ -376,6 +376,8 @@ class RenderingServerDefault : public RenderingServer {
 	FUNC2(light_set_cull_mask, RID, uint32_t)
 	FUNC5(light_set_distance_fade, RID, bool, float, float, float)
 	FUNC2(light_set_reverse_cull_face_mode, RID, bool)
+	FUNC2(light_set_shadow_caster_mask, RID, uint32_t)
+	FUNC1RC(uint32_t, light_get_shadow_caster_mask, RID)
```

Comment:
Does this need to be exposed? Other methods don't have a getter exposed, presumably to avoid pipeline stalls when the method is called.

cc @clayjohn

Reply:
As @Calinou says for getters it is usually preferable to cache scene side and return that rather than query the servers, which can cause a sync and stall the pipeline.